pyomo/contrib/mindtpy/util.py

```diff
@@ -64,13 +75,26 @@ def model_is_valid(solve_data, config):
     if not hasattr(m, 'dual'):  # Set up dual value reporting
         m.dual = Suffix(direction=Suffix.IMPORT)
 
-    # TODO if any continuous variables are multipled with binary ones, need
-    # to do some kind of transformation (Glover?) or throw an error message
+    # TODO if any continuous variables are multiplied with binary ones,
+    #  need to do some kind of transformation (Glover?) or throw an error message
```

Comment:
David, thoughts?

Reply:
This is something we need to work on. Basically, if we identify that there is a binary-continuous product, we can exactly linearize it and make the problem way easier using two constraints. I think that the right place to do this is in a transformation and then call it here, but it hasn't been written yet. This can be solved for the trivial case (where only one binary and one continuous variable compose the constraint kxy <= 0) but can get more complicated once we consider more general cases (binary times a continuous expression, product appearing in the middle of the constraint, ...). This is a convexification technique (a.k.a. making a nonconvex problem into a convex one) and is beyond the scope of this PR.